tests/lib/prepare.sh

```diff
@@ -3,7 +3,8 @@
 set -eux
 
 prepare_classic() {
-    apt install -y ${SPREAD_PATH}/../snapd_*.deb
+    # relying on dpkg as apt(-get) does not support installation from local files in trusty.
+    dpkg -i ${SPREAD_PATH}/../snapd_*.deb || apt-get -f install -y
```

Comment:
Why -f?  Also, let's please make it explicit so it's more clear and sane, leveraging `$SPREAD_SYSTEM`, instead of guessing why these commands are failing.